ansible/roles/omero-user/README.md

```diff
@@ -0,0 +1,81 @@
+OMERO user
+==========
+
+Create OMERO user accounts and groups.
+This will not modify existing users or groups, apart from the user password if `force` is set.
+
+
+Dependencies
+------------
+
+Assumes the `omero-server` role is installed with defaults, if not you will have to set most of the role variables below.
```

Comment:
My thinking was `omero-server` is a pretty hefty role, `omero-user` is relatively minor in comparison, so I didn't want to force the server to be a dependency. For example I could add a `hostname` argument to this role to allow the creation of users remotely, I haven't bothered so far because the driver for this was the IDR where I needed to reset passwords, and the easiest way was to obtain the db connection parameters which belong to `omero-hosts`.